src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -67,7 +67,7 @@ public void RunApplication(Application app)
 
                 DynamoModel.RequestUpdateLoadBarStatus += DynamoModel_RequestUpdateLoadBarStatus;
 
-                splashScreen = new Dynamo.UI.Views.SplashScreen();
+                splashScreen = new Dynamo.UI.Views.SplashScreen(Program.DynamoCorePath);
```

Comment:
This does not seem the right path? FYI: @RobertGlobant20

Reply:
@filipeotero the right path should be something like this:
C:\Users\myuser\AppData\Roaming\Dynamo\Dynamo Core\2.17\
Please verify if this is the expected path.

Reply:
Aaron is right, the path is wrong. But to get the user path, we need to load Dynamo model first. Is it ok @QilongTang?

Reply:
Hi @filipeotero If we load Dynamo Model first, then a few steps from the dynamic splash screen will be done by the time we display it. For now, would you hard code the path in CoreSetup?

Reply:
now its like this: `C:\Users\myuser\AppData\Roaming\Dynamo\Dynamo Core\2.17`